ios/Classes/ReactiveBle/Central.swift

```diff
@@ -11,7 +11,7 @@ typealias ServiceData = [ServiceID: Data]
 final class Central {
 
     typealias StateChangeHandler = (Central, CBManagerState) -> Void
-    typealias DiscoveryHandler = (Central, CBPeripheral, ServiceData, RSSI) -> Void
+    typealias DiscoveryHandler = (Central, CBPeripheral, [String: Any], RSSI) -> Void
```

Comment:
Is `ServiceData` type-alias still used? If not, let's kill it.

Now it would make sense to define `typealias AdvertisementData = [String: Any]`, I guess. I does not make the program more type-safe, but it does make declarations like this more readable / self-explanatory.

Reply:
Done.